src/component/category/Categories.js

```diff
@@ -1,9 +1,13 @@
 import React, { useState } from "react";
+import { MenuList } from "../../constants/MenuConstant";
```

Comment:
react imports in first
other imports in second
gap
our components
gap
constants